### ISSUE TITLE
Attribute error when reading fits files with comments

### ISSUE THREAD

**Issue by hamogu** [atpy/atpy#27]
Hi,

I try to read a fits file with comments, but I get an attribute error. because pyfits (I use version 2.2.2) returns comments as pyfits.cards object, not as string.
This seems to be the behaviuour for the new pyfits 3.0 as well.
I did not have that error before, I don't know where it came from (maybe ouy systems group upgraded pyfits from the old 1.3).

This is the last bit of the error stack:

```
/data/hguenther/Dropbox/code/python/atpy/atpy/decorators.pyc in _auto_fileobj_to_file(read, table, filename, *args, **kwargs)
 72 
 73         # Update filename

 74         filename = output.name
 75 
---> 76     return read(table, filename, *args, **kwargs)

/data/hguenther/Dropbox/code/python/atpy/atpy/fitstable.pyc in read(self, filename, hdu, verbose)
189 
190     for comment in header.get_comment():
--> 191         self.add_comment(comment)
192 
193     if hdu.name:

/data/hguenther/Dropbox/code/python/atpy/atpy/basetable.pyc in add_comment(self, comment)
831         '''
832 
--> 833         self.comments.append(comment.strip())
834         return
835 

AttributeError: 'Card' object has no attribute 'strip'
```

There is an easy fix though: Change in `fitstable.py`:

```
self.add_comment(comment) -> self.add_comment(str(comment))
```

**Comment by hamogu** [comment]
My current problem could be due to a problem with the local pyfits installation we have here.
However, from the pyfits documentation it looks as if the `pyfits.Card` object is returned by `get_comment` for newer versions of pyfits as well.

**Comment by astrofrog** [comment]
This is indeed a bug - I've fixed it in e3eb5742068f9350fc47. Using str() is not a good idea since it will include the "COMMENT" prefix, so I used `comment.value` instead. Since old versions of PyFITS might use `str` instead of `pyfits.Card`, I check for the type of the comment to figure out whether to use `comment.value` or `comment`. Thanks for the report!
